pkgs/applications/audio/giada/default.nix

```
@@ -58,19 +65,16 @@ stdenv.mkDerivation rec {
     libogg
     libvorbis
     libopus
+    libXrandr
   ];
 
-  postPatch = ''
-    local fixup_list=(
-      src/core/kernelMidi.cpp
-      src/gui/elems/config/tabMidi.cpp
-      src/utils/ver.cpp
-    )
-    for f in "''${fixup_list[@]}"; do
-      substituteInPlace "$f" \
-        --replace "<RtMidi.h>" "<${rtmidi.src}/RtMidi.h>"
-    done
-  '';
+  patches = [
+    # Included in next release https://github.com/monocasual/giada/pull/635
```

```suggestion
    # fix fmt type error https://github.com/monocasual/giada/pull/635
```
if I am guessing that correct

---

Wanted to make it clear to remove it on the next update, but added another line along with your suggestion to make it more explicit.